README.md

```diff
@@ -14,15 +14,15 @@ To use leckerli into your website, simply add the following CSS/JavaScript in yo
   <!-- somewhere in the <head /> -->
   <link
     rel="stylesheet"
-    href="//www.unpkg.com/@antistatique/leckerli@1.1/dist/assets/leckerli.min.css"
+    href="//www.unpkg.com/@antistatique/leckerli@1.1/dist/leckerli.min.css"
```

Comment:
We can't change this url without releasing a major version. Existing GTM integration could fail.